docs/programming-languages/javascript-and-node-js.md

```diff
@@ -174,3 +172,40 @@ for details on pre-installed browsers and testing tools on Semaphore.
 [macos-javascript]: https://docs.semaphoreci.com/ci-cd-environment/macos-mojave-xcode-11-image/#javascript-via-node-js
 [docker-env]: https://docs.semaphoreci.com/ci-cd-environment/custom-ci-cd-environment-with-docker/
 [node-docker-image]: https://hub.docker.com/r/semaphoreci/node
+
+## Custom config.js
```

Comment:
This section deals with setting up Database configuration. A better title for this is `Connecting your Node.js application to the test Database`

Reply:
This change might affect the rest of the flow as well.